frontend/talentsearch/resources/js/components/workLocationPreferenceForm/workLocationPreferenceOperations.graphql

```diff
@@ -0,0 +1,6 @@
+mutation createWorkLocationPreference($user: CreateUserInput!) {
```

Comment:
I am not sure! I was thinking its coming in one point when user register(first time). Then I need the ID too.

Reply:
It's true that the first time it will be adding data to empty fields.  However, the `User` model that this mutation is working on will always exist before this mutation runs.  You will never be creating a new user with this mutation.

Reply:
Make sense!

Reply:
I changed it take a look when you can.

Reply:
Done! Please check again.